Fixed typos 669e81368e52b3a5c35a2dc5.md (#56496)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/lab-video-compilation-page/669e81368e52b3a5c35a2dc5.md

# --description--

> Fulfill the user stories below and pass all the tests to complete the lab.

Reviewer: this line should not be changed

```js
assert.equal(document.querySelector('body :first-child').tagName, 'MAIN');
```

> Your main element should be the only child of the body element.

Reviewer: ```suggestion

Your main element should be the only child of the body element.
```
empty lines around code fences are needed

Reviewer: you also removed backticks, do not do that

Author: @ilenia-magoni sorry for the inconvenience from my side , I am new to this

```js
assert.equal(document.querySelector('body').children.length, 2);
```

> You should have an h1 element with the topic of your page inside the main element.

Reviewer: ```suggestion

You should have an h1 element with the topic of your page inside the main element.
```

```js
assert.isAbove(document.querySelector('h1')?.innerText.length, 0);
```

> You should have a paragraph introducing the topic of your page below your h1 element.

Reviewer: ```suggestion

You should have a paragraph introducing the topic of your page below your h1 element.
```

Reviewer: you should not have changed this line, I'm unresolving the conversation

```js
assert.equal(document.querySelector('body :first-child').tagName, 'MAIN');
```

> Your `main` element should be the only child of the `body` element.

Reviewer: these two lines should not be changed

Your main element should be the only child of the body element.

```js
assert.equal(document.querySelector('body').children.length, 2);
```

> You should have an `h1` element with the topic of your page inside the `main` element.

Reviewer: these two lines should not be changed

> You should have an h1 element with the topic of your page inside the main element.

Reviewer: still missing an empty line here, even if you marked this as resolved

You should have an h1 element with the topic of your page inside the main element.

```js
assert.isAbove(document.querySelector('h1')?.innerText.length, 0);
```

> You should have a paragraph introducing the topic of your page below your `h1` element.

Reviewer: these two lines should not be changed

You should have a paragraph introducing the topic of your page below your h1 element.

```js
const p = document.querySelectorAll('p')[0];
assert.equal(p?.previousElementSibling?.tagName, 'H1');
assert.isAbove(p?.innerText.length, 0);
```

> You should have three `section` elements below your first `p` element.

Reviewer: this line should not be changed

> Each section element should contain an h2 element to give a title to that section as its first child.

Reviewer: ```suggestion
Each `section` element should start with an `h2` element that serves as the title for that section.
```

you need to make the required change

    assert.isAbove(section.children[0]?.innerText.length, 0);
}
```

> Each section element should contain a p element to introduce the video content as its second child.

Reviewer: ```suggestion

Each section element should contain a p element to introduce the video content as its second child.
```

}
```

> Each section element should contain a p element to introduce the video content as its second child.

Reviewer: these lines should not be changed

    assert.isAbove(section.children[1]?.innerText.length, 0);
}
```

> Each section element should contain an iframe element as its third child.

Reviewer: ```suggestion

Each section element should contain an iframe element as its third child.
```

}
```

> Each section element should contain an iframe element as its third child.

Reviewer: these lines should not be changed

    assert.equal(section.children[2]?.tagName, 'IFRAME');
}
```

> Your first iframe element should have a src attribute set to a valid video.

Reviewer: ```suggestion

Your first iframe element should have a src attribute set to a valid video.

```

```js
assert.isAbove(document.querySelectorAll('iframe')[0]?.src.length, 0)
```

> Your first iframe element should have a title attribute to describe the embedded content.

Reviewer: ```suggestion

Your first iframe element should have a title attribute to describe the embedded content.
```

```js
assert.isAbove(document.querySelectorAll('iframe')[0]?.title.length, 0)
```

> Your first iframe element should have a height attribute.

Reviewer: ```suggestion

Your first iframe element should have a height attribute.

```

```js
assert.isAbove(document.querySelectorAll('iframe')[0]?.height.length, 0)
```

> Your first iframe element should have a width attribute.
> ```js
> assert.isAbove(document.querySelectorAll('iframe')[0]?.width.length, 0)
> ```

Reviewer: ````suggestion

Your first iframe element should have a width attribute.

```js
assert.isAbove(document.querySelectorAll('iframe')[0]?.width.length, 0)
```
````

Your second iframe element should have a src attribute set to a valid video.
```js
assert.isAbove(document.querySelectorAll('iframe')[1]?.src.length, 0)
```

> Your second iframe element should have a title attribute to describe the embedded content.

Reviewer: ```suggestion

Your second iframe element should have a title attribute to describe the embedded content.

```

```js
assert.isAbove(document.querySelectorAll('iframe')[1]?.title.length, 0)
```

> Your second iframe element should have a height attribute.

Reviewer: and so on

}
```

> Your first iframe element should have a src attribute set to a valid video.

Reviewer: this line should not be changed

```js
assert.isAbove(document.querySelectorAll('iframe')[0]?.src.length, 0)
```

> Your first iframe element should have a title attribute to describe the embedded content.

Reviewer: this line should not be changed

```js
assert.isAbove(document.querySelectorAll('iframe')[0]?.title.length, 0)
```

> Your first iframe element should have a height attribute.

Reviewer: this line should not be changed

```js
assert.isAbove(document.querySelectorAll('iframe')[0]?.height.length, 0)
```

> Your first iframe element should have a width attribute.

Reviewer: this line should not be changed

```js
assert.isAbove(document.querySelectorAll('iframe')[0]?.width.length, 0)
```

> Your second iframe element should have a src attribute set to a valid video.

Reviewer: this line should not be changed

```js
assert.isAbove(document.querySelectorAll('iframe')[1]?.src.length, 0)
```

> Your second iframe element should have a title attribute to describe the embedded content.

Reviewer: this line should not be changed

```js
assert.isAbove(document.querySelectorAll('iframe')[1]?.title.length, 0)
```

> Your second iframe element should have a height attribute.

Reviewer: this line should not be changed

```js
assert.isAbove(document.querySelectorAll('iframe')[1]?.height.length, 0)
```

> Your second iframe element should have a width attribute.

Reviewer: this line should not be changed

```js
assert.isAbove(document.querySelectorAll('iframe')[1]?.width.length, 0)
```

> Your third iframe element should have a src attribute set to a valid video.

Reviewer: this line should not be changed

```js
assert.isAbove(document.querySelectorAll('iframe')[2]?.src.length, 0)
```

> Your third iframe element should have a title attribute to describe the embedded content.

Reviewer: this line should not be changed

```js
assert.isAbove(document.querySelectorAll('iframe')[2]?.title.length, 0)
```

> Your third iframe element should have a height attribute.

Reviewer: this line should not be changed

```js
assert.isAbove(document.querySelectorAll('iframe')[2]?.height.length, 0)
```

> Your third iframe element should have a width attribute.

Reviewer: this line should not be changed

```js
assert.isAbove(document.querySelectorAll('iframe')[2]?.width.length, 0)
```

# --seed--

> # --seed-contents--

Reviewer: these lines should not be changed
